.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "java.configuration.updateBuildConfiguration": "disabled"
```

Comment:
This likely needs to be explicitly removed - it shouldn't be source controlled given the .gitignore.